src/core/MOM_barotropic.F90

```diff
@@ -2743,30 +2727,30 @@ subroutine btstep_find_Cor(q, DCor_u, DCor_v, amer, bmer, cmer, dmer, azon, bzon
   !$OMP parallel do default(shared)
   do J=jsvf-1,jevf ; do i=isvf-1,ievf+1
     if (CS%Sadourny) then
-      amer(I-1,j) = DCor_u(I-1,j) * q(I-1,J)
-      bmer(I,j) = DCor_u(I,j) * q(I,J)
-      cmer(I,j+1) = DCor_u(I,j+1) * q(I,J)
-      dmer(I-1,j+1) = DCor_u(I-1,j+1) * q(I-1,J)
+      f_4_v(1,i,J) = DCor_u(I-1,j) * q(I-1,J)
+      f_4_v(2,i,J) = DCor_u(I,j) * q(I,J)
+      f_4_v(4,i,J) = DCor_u(I,j+1) * q(I,J)
+      f_4_v(3,i,J) = DCor_u(I-1,j+1) * q(I-1,J)
```

Comment:
here it looks like `amer(I-1,j)` or northeast is being replaced by `f_4_v(1,i,j)` which is labeled southwest. 
I am i understanding correctly that this is because of the change in the horizontal index (I-1,j) to (i,j)?